src/index.html

```diff
@@ -4,10 +4,349 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="icon" href="images/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__content">
+          <div class="header__contacts">
+            <a class="header__contacts header__contacts--phone-link" href="tel:+1 234 555-55-55">
+              <img
+                src="/images/Phone.png"
+                alt="logo"
+                class="header__contacts-img"
+              >
+            </a>
+            <a
+              class="header__phone"
+              href="tel:+1 234 555-55-55"
+            >
+              +1 234 555-55-55
+            </a>
+            <img
+              src="/images/logo.png"
+              alt="logo"
+              class="header__logo"
+            >
+            <a
+              class="header__order"
+              href="mailto:https://github.com/TishchenkoSergey"
+            >
+              call to order
+            </a>
+            <a class="header__contacts header__contacts-img">
+              <img
+                src="/images/Menu.png"
+                alt="logo"
+                class="header__contacts-image"
+              >
+            </a>
+          </div>
+          <h1 class="header__title">Miami Condo Kings</h1>
+        </div>
+
+        <nav class="header__nav">
+          <ul class="header__nav-list">
+            <li class="header__nav-item">
+              <a href="#" class="header__nav-link header__nav-link--is-active">home</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#about" class="header__nav-link">about&nbsp;us</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#gallery" class="header__nav-link">gallery</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#consultation" class="header__nav-link">consultation</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#testimonials" class="header__nav-link">testimonials</a>
+            </li>
+            <li class="header__nav-item">
+              <a href="#contacts" class="header__nav-link">contacts</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="section" id="about">
+        <div class="info">
+          <h2 class="info__title">Miami best Real estate
+            <a class="info__link info__link-decoration" href="#">here</a>
+          </h2>
```

Comment:
```suggestion
          <h2 class="info__title">
            Miami best Real estate
            <a class="info__link info__link-decoration" href="#">here</a>
          </h2>
```
Если переносим контент тега на новую строку, то переносим все, а не какую-то часть